src/panels/lovelace/cards/hui-media-control-card.ts

```diff
@@ -87,103 +123,159 @@ export class HuiMediaControlCard extends LitElement implements LovelaceCard {
     return html`
       <ha-card>
         <div
-          class="ratio ${classMap({
+          class="background ${classMap({
             "no-image": !this._image,
           })}"
         >
+          <div
+            class="color-block"
+            style="background-color: ${this.backgroundColor}"
+          ></div>
+          ${!this._image
+            ? ""
+            : html`
+                <div
+                  class="color-gradient"
+                  style="background-image: linear-gradient(to right, ${this
+                    .backgroundColor}, transparent);width: ${this
+                    ._cardWidth}px;"
+                ></div>
+              `}
           <div
             class="image"
-            style="background-image: url(${this.hass.hassUrl(picture)})"
+            style="background-image: url(${this.hass.hassUrl(
+              picture
+            )}); padding-left: ${!this._image
+              ? "50%"
+              : `${this._cardWidth}px`};"
           ></div>
+        </div>
+        <div
+          class="player ${classMap({
+            "no-image": !this._image,
+            narrow: this._narrow,
+            "very-narrow": this._veryNarrow,
+          })}"
+          style="color: ${this.foregroundColor}"
+        >
+          <div class="top-info">
+            <div>
+              ${!stateObj.attributes.icon
+                ? ""
+                : html`
+                    <ha-icon
+                      class="icon"
+                      .icon="${stateObj.attributes.icon}"
+                    ></ha-icon>
+                  `}
+              ${this._config!.name ||
+                computeStateName(this.hass!.states[this._config!.entity])}
+            </div>
+            <div>
+              <paper-icon-button
+                icon="hass:dots-vertical"
+                class="more-info"
+                @click=${this._handleMoreInfo}
+              ></paper-icon-button>
+            </div>
+          </div>
           <div
-            class="caption ${classMap({
-              unavailable: stateObj.state === UNAVAILABLE,
-            })}"
+            class="title-controls"
+            style="padding-right: ${this._cardWidth - 16}px"
```

Comment:
Can this be done with `calc`.
```suggestion
            style="padding-right: calc(100% - 16px)"
```